### PR TITLE
Updated breadcrumbs

### PR DESCRIPTION
**Pull request by joshsadam**
## Description of changes
Cleans up the style of the breadcrumbs to better reflect the current IRIDA ant design layouts.  Replaced just displaying the identifier for an item with the actual label.

## Related issue
#101 

## Checklist
Things for the developer to confirm they've done before the PR should be accepted:

* [x] CHANGELOG.md (and UPGRADING.md if necessary) updated with information for new change.
* [x] Tests added (or description of how to test) for any new features.
~~* [ ] User documentation updated for UI or technical changes.~~

### CODE REVIEW

File: src/main/webapp/resources/js/components/Header/Breadcrumbs.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from "react";
```
Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Header/Breadcrumbs.jsx]: File comments

Comment (deepsidhu85) [inline src/main/webapp/resources/js/components/Header/Breadcrumbs.jsx]: Add file comments.

Comment (joshsadam) [reply src/main/webapp/resources/js/components/Header/Breadcrumbs.jsx]: Added in 54c034c4456ff3dd88a2ee4c67df4db59d06e822

Comment (deepsidhu85) [reply src/main/webapp/resources/js/components/Header/Breadcrumbs.jsx]: Awesome thanks

**Pull request merged**